### PR TITLE
Feature/control individual log levels

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**

### CODE REVIEW

File: docs/node/node_config.yaml [hunk 1]
```diff
@@ -165,6 +165,30 @@ application:
...
+        level: warning
+
+  # some additional debug flags
+  debug:
+
+    # sets the socketio client into debug mode
+    socketio: true
+
+    # sets the Flask app used for the LOCAL proxy service into debug mode
+    proxy_server: true
```
Comment (bartvanb) [inline docs/node/node_config.yaml:189]: Shouldn't we have these by default at false?

File: docs/server/yaml/server_config.yaml [hunk 2]
```diff
@@ -67,6 +67,31 @@ environments:
...
+
+    # some additional debug flags
+    debug:
+
+      # sets the socketio server into debug mode
+      socketio: true
+
+      # sets the Flask app in debug mode
+      flask: true
+
```
Comment (bartvanb) [inline docs/server/yaml/server_config.yaml:94]: same as above

File: docs/node/node_config.yaml (hunk 1 above)
Comment (bartvanb) [reply docs/node/node_config.yaml:189]: And would it make sense to add 1 indent? And should we list all the options for packages? (I think it's just 'flask' missing right?)

**Review by bartvanb — CHANGES_REQUESTED** [review]
Good improvement, just a few minor comments

File: docs/node/node_config.yaml (hunk 1 above)
Comment (frankcorneliusmartin) [reply docs/node/node_config.yaml:189]: Made them by default False. Identation does not make sense to me as the logging session refers to the configuration of the  python logger.

**Pull request merged**